open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/impl/specialkey/EllipseSemiMinorAxisKey.java

```diff
@@ -48,4 +78,26 @@ public EllipseSemiMinorAxisKey(Class<? extends Length> semiMinorUnit)
     {
         return (Class<? extends Length>)getKeyUnit();
     }
+
+    /**
+     * Attempts to detect the length unit from the column name.
+     *
+     * @param columnName the name of the column to inspect
+     * @return the detected length unit, or null if it couldn't be detected
+     */
+    public static Class<? extends Length> detectUnit(String columnName)
+    {
+        return EllipseSemiMajorAxisKey.detectUnit(columnName);

```

Comment:
It's a bit strange to re-use the semi-major unit detector... maybe a future task would be to pull that out to a utility that could be called by both?